install.py

```diff
@@ -879,12 +877,13 @@ def driver():
         help="Maximum number of fields that Legate will support",
     )
     parser.add_argument(
-        "--gasnet",
-        dest="gasnet",
-        action="store_true",
+        "--network",
```

Comment:
CC @trxcllnt @jefflarkin: Note this modification to an `install.py` flag, that might conflict with your changes.